_sc2data/Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -589,6 +593,10 @@
         <WhichPlayer Value="Caster"/>
         <Value value="AP_HaveHotSMonarchBlades"/>
     </CValidatorPlayerRequirement>
+    <CValidatorUnitCompareBehaviorCount id="AP_NotFungalGrowthed">
+        <Value value="1"/>
+        <Behavior value="AP_FungalGrowth"/>
+    </CValidatorUnitCompareBehaviorCount>

```

Comment:
Patch this validator instead (from Liberty) using the pattern https://github.com/Ziktofel/Archipelago/wiki/Porting-units-to-ArchipelagoPlayer-SC2Mod#validators:

```xml
    <CValidatorUnitCompareBehaviorCount id="NotFungalGrowthed">
        <Behavior value="FungalGrowth"/>
        <Value value="1"/>
    </CValidatorUnitCompareBehaviorCount>
```